### DISCUSSION TITLE
Using "vendoring" to include another repository (written in GoLang) to my own.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#127517] [Repositories]
### Select Topic Area

Product Feedback

### Body

Greetings!

I am relatively new to github so apologies if this is just basic knowledge. I would like to include another repository into my work, but I am not sure how to go about it. All I do understand is that it should look something similar to what is shown in the picture below.

![image](https://github.com/community/community/assets/156294428/2a7b6a23-35d8-4ece-9f33-509e5abf3bd3)

For reference, here is the source of the picture:
https://github.com/lheijst/rtldavis/tree/master/vendor/github.com/jpoirier

I intend to add another github repository, which is written in Go language similar to the one from the source, and use it as a dependency. To be even more specific, the repository I want to add is found in the link below.
https://github.com/hztools/go-sdr/tree/main/pluto

Any help will be much appreciated. Thank you very much!

**Comment by ebndev** [comment]
Hi @imca-potato, thanks for being a part of the GitHub Community!

I've gone ahead and moved this post to our Repositories category, as this topic might be more relevant there and hopefully that might help someone give you a nudge in the right direction.
